src/core/entities/MRVideo.js

```diff
@@ -0,0 +1,211 @@
+import * as THREE from 'three';
+
+import { MRDivEntity } from 'mrjs/core/MRDivEntity';
+
+import { mrjsUtils } from 'mrjs';
+
+/**
+ * @class MRVideo
+ * @classdesc Base html video represented in 3D space. `mr-video`
+ * @augments MRDivEntity
+ */
+export class MRVideo extends MRDivEntity {
```

Comment:
@hanbollar you're currently working on the fix for `object-fit`, MRVideo should be treated the same as MRImage. Perhaps we can extract their common features out into MRMedia? that wouldn't need to be exposed as a tag itself, but might make it easy for us to apply the same responsiveness logic to both.

Reply:
yea we can def do something like that the same way we're doing mrtext --> mrtextfield && mrtextarea

i'll make an open todo to do that after the mrimage fix 👍

Reply:
noting: #437